modules/rosa-classic-cluster-terraform-file-creation.adoc

```diff
@@ -83,17 +81,11 @@ locals {
 resource "time_sleep" "wait_60_seconds" {
   count = var.create_vpc ? 1 : 0
   depends_on = [module.vpc]
-  create_duration = "60s"
 }
 
 module "rosa-classic" {
   source                 = "terraform-redhat/rosa-classic/rhcs"
   version                = "1.5.0"
-  cluster_name           = local.cluster_name
-  openshift_version      = var.openshift_version
```

Comment:
These two options are required for the module to work. Seems a bit weird to get rid of the other options as well